courses/api.py

```diff
@@ -582,6 +582,14 @@ def sync_course_runs(runs):
             run.enrollment_start = course_detail.enrollment_start
             run.enrollment_end = course_detail.enrollment_end
             run.is_self_paced = course_detail.is_self_paced()
+            # Only sync the date if it's set in edX, Otherwise set it to course's end date
+            if course_detail.certificate_available_date:
+                run.certificate_available_date = (
+                    course_detail.certificate_available_date
+                )
+            else:
+                run.certificate_available_date = course_detail.end
```

Comment:
@pdpinch During the sync process, if there is no certificate available date set in edX or the certificate display behaviour is turned off I set `certificate_available_date` to the course run's end date. let me know if that makes sense.

Reply:
@pdpinch Do you have any thoughts on this before I merge this?

Reply:
That makes sense for instructor-paced courses, since that's what we do already. 

I assume that setting the `certificate_available_date` has no affect on the behavior of self-paced courses.

Reply:
Thanks for sharing your thoughts!

>I assume that setting the certificate_available_date has no affect on the behavior of self-paced courses.

That's Correct, I'll rebase and merge in a bit.